tools/azure-plugin/generator/Azure.Generator/src/Azure.Generator.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <PackageId>AzureClientPlugin</PackageId>
+    <Version>1.0.0-beta.1</Version>
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
+	<!-- Below is workaround to make MSBuild pass -->
+	<RequiredTargetFrameworks>net8.0</RequiredTargetFrameworks>
+	<TargetFrameworks>net8.0</TargetFrameworks>
```

Comment:
This takes precedence over the `TargetFramework` that you added earlier.   No reason to have both.  Let's use this one and follow the patterns in this repository.

Reply:
removed